MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -1588,385 +1544,471 @@ private void initComponents() {
+                gridBagConstraints = new java.awt.GridBagConstraints();
+                gridBagConstraints.gridx = 1;
+                gridBagConstraints.gridy = 0;
+                gridBagConstraints.weightx = 0.0;
+                gridBagConstraints.weighty = 0.0;
+                gridBagConstraints.fill = java.awt.GridBagConstraints.NONE;
+                gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+                panType.add(txtType, gridBagConstraints);
+                txtSalaryBase[i] = txtType;
+                panAllTypes.add(panType);
+            }
+            JScrollPane scrSalaryBase = new JScrollPane(panAllTypes);
+            scrSalaryBase.setBorder(BorderFactory.createTitledBorder("Base Salaries"));
+            scrSalaryBase.setOpaque(false);
+            scrSalaryBase.setPreferredSize(new Dimension(200, 200));
+            gridBagConstraints = new java.awt.GridBagConstraints();
+            gridBagConstraints.gridx = 0;
+            gridBagConstraints.gridy = 2;
+            gridBagConstraints.weightx = 1.0;
+            gridBagConstraints.weighty = 1.0;
+            gridBagConstraints.fill = java.awt.GridBagConstraints.BOTH;
+            gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+            panSalary.add(scrSalaryBase, gridBagConstraints);
 
-        gridBagConstraints = new java.awt.GridBagConstraints();
-        gridBagConstraints.gridx = 3;
-        gridBagConstraints.gridy = 0;
-        gridBagConstraints.gridheight = 23;
-        gridBagConstraints.weightx = 1.0;
-        gridBagConstraints.weighty = 1.0;
-        gridBagConstraints.fill = java.awt.GridBagConstraints.BOTH;
-        gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
-        panPersonnel.add(panSalary, gridBagConstraints);
+            gridBagConstraints = new java.awt.GridBagConstraints();
+            gridBagConstraints.gridx = 3;
+            gridBagConstraints.gridy = 0;
+            gridBagConstraints.gridheight = 23;
+            gridBagConstraints.weightx = 1.0;
+            gridBagConstraints.weighty = 1.0;
+            gridBagConstraints.fill = java.awt.GridBagConstraints.BOTH;
+            gridBagConstraints.anchor = java.awt.GridBagConstraints.NORTHWEST;
+            panPersonnel.add(panSalary, gridBagConstraints);
 
-        JScrollPane scrollPersonnel = new JScrollPane(panPersonnel);
-        scrollPersonnel.setPreferredSize(new java.awt.Dimension(500, 400));
+            JScrollPane scrollPersonnel = new JScrollPane(panPersonnel);
+            scrollPersonnel.setPreferredSize(new java.awt.Dimension(500, 400));
 
-        tabOptions.addTab(resourceMap.getString("panPersonnel.TabConstraints.tabTitle"), scrollPersonnel); // NOI18N
+            tabOptions.addTab(resourceMap.getString("panPersonnel.TabConstraints.tabTitle"), scrollPersonnel);
+        } catch (Exception e) {
+            MekHQ.getLogger().error(getClass(), "PersonnelTab", e);
```

Comment:
What sort of exceptions are we trying to catch that we can't expect?

Reply:
Found it super useful while testing, but forgot to remove it after. Removing